src/blog/2024/07/customise-theming-in-your-dashboards.md

```diff
@@ -0,0 +1,215 @@
+
+### Right Side of the Header
+
+To render elements on the right side of the header, you can use the empty div element having the `#app-bar-actions` ID, in which we can add elements.
+
+![Right Side Area](./images/right-side-area.png){data-zoomable}
+
+The `#app-bar-actions` currently has a width of 0px. As elements are added, this container will dynamically adjust its width based on the width of its children. Items will align from right to left. Similar to the left side, adding many elements in succession (using `display: flex`) may cause them to extend beyond the width of the page view.
+
+#### Example: Adding logged in user profile
+
+In this section, we will add the user profile of the currently logged-in user to the right side of the header. Make sure you have installed "@flowfuse/node-red-dashboard-2-user-addon" via the palette manager and enabled [FlowFuse User Authentication](/docs/user/instance-settings/#flowfuse-user-authentication). Each message emitted by the Dashboard 2.0 widget will include the logged-in user information under `msg._client.user`. Additionally the [setup object](https://dashboard.flowfuse.com/contributing/guides/state-management.html#setup-store) will also contain this information under `setup.socketio.auth.user`.
+
+1. Drag the ui-template widget onto the canvas.
+2. Select the correct scope for that widget to render.
+3. Select the correct page or UI in which you want to render the element.
+3. Paste the same Vue snippet given below into the `ui-template` widget:
+
+```js
+<template>
+    <!-- Teleporting user info to #app-bar-actions, which is the ID of the action bars' right corners area -->
+    <Teleport v-if="loaded" to="#app-bar-actions">
+        <div class="user-info">
+            <!-- Displaying user image -->
+            <img :src="setup.socketio.auth.user.image" />
+            <!-- Greeting the user -->
+            <span>Hi, {{ setup.socketio.auth.user.name }}</span>
+        </div>
+    </Teleport>
+</template>
+
+<script>
+export default {
+    data() {
+        return {
+            // Flag to indicate if the component is loaded
+            loaded: false
+        };
+    },
+    mounted() {
+        // This function is called when the component is inserted into the DOM.
+        // Setting loaded to true here ensures the component is ready to access #app-bar-actions,
+        // as it's now part of the same DOM structure.
+        // Accessing it before mounted() would cause an error because the component wouldn't be initialized in the DOM yet.
+        this.loaded = true; // Setting loaded to true to indicate that the component has been mounted successfully
+    }
+}
+</script>
+
+<style>
+/* Styling for user info display */
+.user-info {
+    display: flex;
+    align-items: center;
+    gap: 8px;
+}
+/* Styling for user avatar image*/
+.user-info img {
+    width: 24px;
+    height: 24px;
+}
+</style>
+```
+
+![Right Side Area](./images/user-profile.png){data-zoomable}
```

Comment:
move this image to be straight after the "Adding logged in user profile" header so it's the first thing the user sees in the section.

Also, add a caption and improve on the alt text